aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -192,64 +232,48 @@ Confirm that the published app for deployment isn't running. Files in the *publi
   }
   ```
 
-### Web Deploy with Visual Studio
-
-See the [Visual Studio publish profiles for ASP.NET Core app deployment](xref:host-and-deploy/visual-studio-publish-profiles#publish-profiles) topic to learn how to create a publish profile for use with Web Deploy. If the hosting provider supplies a Publish Profile or support for creating one, download their profile and import it using the Visual Studio **Publish** dialog.
-
-![Publish dialog page](index/_static/pub-dialog.png)
-
-### Web Deploy outside of Visual Studio
-
-[Web Deploy](/iis/publish/using-web-deploy/introduction-to-web-deploy) can also be used outside of Visual Studio from the command line. For more information, see [Web Deployment Tool](/iis/publish/using-web-deploy/use-the-web-deployment-tool).
-
-### Alternatives to Web Deploy
-
-Use any of several methods to move the app to the hosting system, such as Xcopy, Robocopy, or PowerShell. Visual Studio users may use the [Publish Samples](https://github.com/aspnet/vsweb-publish/blob/master/samples/samples.md).
-
-## Browse the website
-
-![The Microsoft Edge browser has loaded the IIS startup page.](index/_static/browsewebsite.png)
-
 ## Data protection
 
-Data Protection is used by several ASP.NET middlewares, including those used in authentication. Even if Data Protection APIs aren't called from user's code, Data Protection should be configured with a deployment script or in user code to create a persistent key store. If data protection isn't configured, the keys are held in memory and discarded when the app restarts.
+The [ASP.NET Core Data Protection stack](xref:security/data-protection/index) is used by several ASP.NET Core [middlewares](xref:fundamentals/middleware/index), including middleware used in authentication. Even if Data Protection APIs aren't called by user code, data protection should be configured with a deployment script or in user code to create a persistent cryptographic [key store](xref:security/data-protection/implementation/key-management). If data protection isn't configured, the keys are held in memory and discarded when the app restarts.
 
-If the keyring is stored in memory when the app restarts:
+If the key ring is stored in memory when the app restarts:
 
-* All forms authentication tokens are invalidated. 
+* Authentication tokens are invalidated. 
```

Comment:
Depends what they are :) If they were JWT they won't be. So "All cookie based authentication tokens issued will be invalidated."